packages/ffe-buttons-react/src/ButtonGroup.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { bool, string } from 'prop-types';
+import classNames from 'classnames';
+
+const ButtonGroup = ({ className, condensed, ...rest }) =>
+    (
+        <div
+            className={classNames(
+                'ffe-button-group',
+                { 'ffe-button-group--thin': condensed },
+                className
+            )}
+            {...rest}
+        />
+    );
+
+ButtonGroup.propTypes = {
+    /** Extra class name applied in addition to the FFE classes */
+    className: string,
+    /** Condensed modifier. Use in condensed designs */
+    condensed: bool,
```

Comment:
Denne condensed-propen er vel ikke bare for condensed designs - det er vel basically når man vil ha en button group uten marginene?

Reply:
Jeg så på det som samme konsept egentlig, og irriterte meg litt over at den modifieren het `thin`. Men er kanskje jeg som blander sammen to ulike use case?

Reply:
Jeg tror det er to forskjellige konsept. Jeg vet ærlig talt ikke helt hvor dette konseptet med spacingen til en knappegruppe kom fra - kanskje den egentlig burde miste marginen sin (siden det tross alt ikke handler om layout)?

Reply:
OK, dette har ikke så mye med den opprinnelige bugen å gjøre. Hva vil du helst at jeg skal gjøre? Fjerne `Use in condensed designs`? Rename til `thin` ? Fjerne hele komponenten?

Reply:
Hm. Vi har ingen gode standarder på tvers. Enten kall den `thin` eller `noMargin` kanskje? Jeg er good med hva enn du velger :+1: